recipes/catch2/2.x.x/conanfile.py

```diff
@@ -51,3 +51,4 @@ def package_id(self):
 
     def package_info(self):
         self.cpp_info.builddirs = [os.path.join("lib", "cmake", "Catch2")]
+        self.cpp_info.name = "Catch2"
```

Comment:
I think it's only for CMake, right? (e.g. not for pkg-config and others)
then it's better to use:
```
self.cpp_info.names["cmake_find_package"] = "Catch2"
self.cpp_info.names["cmake_find_package_multi"] = "Catch2"
```

Reply:
Agreed. fixed